rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc

```diff
@@ -26,63 +26,72 @@ Create and document the environment for the automation of processes.
-* Global variables and credentials can be linked to activity parameters defined in automations. The activity parameters for a process are defined in RPA Builder. Linking global variables or credentials overwrites the initial value of the linked activity parameters in a run configuration.
+If you want to use RPA Builder to xref:rpa-builder::toolbox-variable-handling-credentials-for-oauth.adoc#generating-an-oauth-refresh-token[generate a refresh token for your OAuth credentials], ask an administrator to xref:usermanagement-manage.adoc#assign-users-rpa-builder-access[assign you RPA Builder access].
 
 == Prepare Projects
 
-Projects are assigned to categories. A category describes the environment of a process in a few keywords.
+Project managers organize projects into categories. A category describes the environment of a process in a few keywords.
 
-Applications that are used in processes are assigned to the process and to bots which can use them. A process can be deployed on a bot which has assigned the required processes.
+Administrators list available applications and assign them to bots which can access them. Citizen technologists assign used applications to processes. You can deploy a process on a bot which has assigned the required processes.
 
-Global variables and credentials are used for values that are the same in multiple processes and which frequently change. This way you can reduce the time involved and the error rate in the maintenance of your processes. You can link global variables and credentials anywhere that a globe symbol is displayed.
+You use global variables and credentials for values that are the same in multiple processes and which frequently change. This way you can reduce the time involved and the error rate in the maintenance of your processes. You can link global variables and credentials anywhere that a globe symbol is displayed.
 
 === Manage Categories
 
-Projects are assigned to categories. A category describes the environment of a process in a few keywords.
+Project managers organize projects into categories. A category describes the environment of a process in a few keywords, for example `Customer Support Mystical University` or `Finance`.
 
-A project can only be assigned to a single category. First create a standard category, so you can start with process automation quickly. You can change the category of a process later. After you have automated the first projects, it makes sense to create an archive category and move completed projects there.
+A project can only belong to a single category. First, create a standard category, so you can start with process automation quickly. You can change the category of a process later. After you have automated the first projects, create an archive category and move completed projects there.
 
 ==== Create a Category
 
-Create categories to group processes. Assign a process to a category during evaluation or project creation.
+Create categories to group processes.
```

Comment:
Suggestion: Create categories to group processes and then assign a proces to a category during evaluation or project creation.

Reply:
Replaced with 'Create categories for the organization of processes.' because the person creating the categories is not the same person assigning them.